### PR TITLE
Image-Input-Component

### PR DESCRIPTION
**Pull request by prajapati-priyanka**
-add image and input component.
-it is not responsive, will make it in the end.

csb link: https://codesandbox.io/s/stupefied-volhard-rcedb

### CODE REVIEW

File: component/input/input.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+input {
+  display: block;
+  padding: 0.8rem 1rem;
+  border-radius: 0.5rem;
+  border: none;
+  width: 25%;
+  margin: 0.5rem 0;
+}
+.input-error {
```
Comment (shaik-saad) [inline component/input/input.css:12]: You can add an empty line after the closing flower brackets, it increases the readability.

**Review by shaik-saad — APPROVED** [review]
Added a suggestion, clean code Priyanka!, you ca go ahead with the merge.

File: component/input/input.css (hunk 1 above)
Comment (prajapati-priyanka) [reply component/input/input.css:12]: Okay, Shaik. I will remember it. Thank you.

**Review by AtulPant2704 — APPROVED** [review]

**Review by Husain01 — APPROVED** [review]
Looks good to go. Nice work 🎉

**Pull request merged**